crates/db_views/src/structs.rs

```diff
@@ -1174,16 +1138,14 @@ pub(crate) struct SearchCombinedViewInternal {
   pub post_tags: PostTags,
   // // Comment-specific
   pub comment: Option<Comment>,
-  pub comment_counts: Option<CommentAggregates>,
   pub comment_saved: Option<DateTime<Utc>>,
   pub my_comment_vote: Option<i16>,
   // // Community-specific
   pub community: Option<Community>,
-  pub community_counts: Option<CommunityAggregates>,
   pub community_blocked: bool,
   pub subscribed: SubscribedType,
   // Person
-  pub item_creator_counts: Option<PersonAggregates>,
+  pub item_creator_counts: Option<Person>,
```

Comment:
Can remove this, since `item_creator` is already below.

Search this file for `count` , to see if there are any others.

Reply:
Fixed